src/legendoptics/pmts.py

```diff
@@ -255,7 +279,7 @@ def pyg4_pmt_attach_photocathode_reflectivity(mat, reg) -> None:
         mat.addVecPropertyPint("REFLECTIVITY", wvl.to("eV"), refl)
 
 
-def pyg4_pmt_attach_photocathode_efficiency(mat, reg) -> None:
+def pyg4_pmt_attach_photocathode_efficiency(mat, reg, name="etl9350") -> None:
```

Comment:
```suggestion
def pyg4_pmt_attach_photocathode_efficiency(
    mat,
    reg,
    name: Literal["etl9350", "gerda", "r7081", "l1000"] = "etl9350"
) -> None:
```

and add `from typing import Literal` at the top